packages/picasso.js/src/core/data/extractor.js

```diff
@@ -42,7 +42,12 @@ export default function extract(dataConfig, data = {}, opts = {}) {
           if (!s) {
             return;
           }
-          extracted.items.push(...s.extract(cfg));
+          const tmp = s.extract(cfg);
+          if (tmp.length < 100000) {
```

Comment:
Where is the 100k value coming from?

Reply:
https://stackoverflow.com/questions/14723848/push-multiple-elements-to-array

Reply:
Yes, but my point was more that it is an arbitrary and is it a magic number. We should at least create a variable with a descriptive name that holds the number and possibly add some comment as to the merit for choosing 100k.

Reply:
Great!